### PR TITLE
[Enhancement] GR8 V1 Update control for Network diagram and high level design doc

### PR DESCRIPTION
**Pull request by dutt0**
## Overview/Summary

GR8 V1 Update control for Network diagram and high level design doc

## This PR fixes/adds/changes/removes

This PR is to update control for Network diagram and high level design doc for GR8 V1

### Breaking Changes
N/A

## Testing Evidence
image

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/ssc-spc-ccoe-cei/azure-guardrails-solution-accelerator/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/ssc-spc-ccoe-cei/azure-guardrails-solution-accelerator/issues)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/ssc-spc-ccoe-cei/azure-guardrails-solution-accelerator/tree/main)
- [x] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.
- [x] Ensure PowerShell module versions have been updated (manually or with the ./tools/Update-ModuleVersions.ps1 script)

### CODE REVIEW

File: src/Guardrails-Common/GR-Common.psm1 [hunk 1]
```diff
@@ -388,7 +388,8 @@ function Check-DocumentExistsInStorage {
     else {
         # no blob with the name $attestationFileName was found in the specified storage account
         $docMissing = $true
-        $commentsArray += $msgTable.procedureFileNotFound -f $ItemName, $DocumentName[0], $ContainerName, $StorageAccountName
+        # $commentsArray += $msgTable.procedureFileNotFound -f $ItemName, $DocumentName[0], $ContainerName, $StorageAccountName
```
Comment (singhgss) [inline src/Guardrails-Common/GR-Common.psm1:391]: Any reason to keep it as comment @dutt0

Comment (dutt0) [reply src/Guardrails-Common/GR-Common.psm1:391]: no :)

**Review by singhgss — APPROVED** [review]

**Review by alalvi00 — CHANGES_REQUESTED** [review]
Remove the extra line from the sign-scripts-development.yml and delete the commented out code

File: .github/workflows/sign-scripts-development.yml
[the remark this answers is not in the document]
Comment (dutt0) [reply .github/workflows/sign-scripts-development.yml:1]: Its already updated in the code

**Review by alalvi00 — APPROVED** [review]

**Pull request merged**